ouroboros-network/test/Test/Mux.hs

```diff
@@ -76,18 +106,41 @@ instance Mx.ProtocolEnum TestProtocols2 where
   toProtocolEnum 3 = Just BlockFetch2
   toProtocolEnum _ = Nothing
 
+instance Mx.MiniProtocolLimits TestProtocols2 where
+  maximumMessageSize ChainSync2   = defaultMiniProtocolLimit
+  maximumMessageSize BlockFetch2  = defaultMiniProtocolLimit
```

Comment:
why not just `maximuMessageSize _ = defaultMiniProtocolLimit`, the same for `maximumIngressQueue`.

Reply:
Will address.